Dockerfile-e2e

```diff
@@ -1,6 +1,6 @@
 FROM 113355358853.dkr.ecr.eu-west-1.amazonaws.com/ho/hmpo/lev/fro/cypress-testing:8c64f6e43a7e98ec1a25a1bf2bf7d830ef151e56
 # Use image below for local testing. Proxies also require removal.
-# FROM cypress/included:cypress-12.17.1-node-18.16.0-chrome-114.0.5735.133-1-ff-114.0.2-edge-114.0.1823.51-1
+#FROM cypress/included:cypress-12.17.1-node-18.16.0-chrome-114.0.5735.133-1-ff-114.0.2-edge-114.0.1823.51-1
```

Comment:
I appreciate that this was previously commented out, but is there a case where this is uncommented? If not, can this be removed?